src/components/ChoiceGroup/ChoiceGroup.test.js

```diff
@@ -185,18 +209,27 @@ describe('ChoiceGroup', () => {
           <Checkbox value="mugatu" />
         </ChoiceGroup>
       )
-      const input = wrapper.find(Checkbox).at(0).find('input')
-      const input2 = wrapper.find(Checkbox).at(1).find('input')
-      const input3 = wrapper.find(Checkbox).at(2).find('input')
+      const input = wrapper
+        .find(Checkbox)
+        .at(0)
+        .find('input')
+      const input2 = wrapper
+        .find(Checkbox)
+        .at(1)
+        .find('input')
+      const input3 = wrapper
+        .find(Checkbox)
+        .at(2)
+        .find('input')
 
       input.simulate('change', { target: { checked: true } })
-      expect(spy).toHaveBeenCalledWith(['derek'])
+      expect(spy).toHaveBeenCalledWith('derek')
 
       input2.simulate('change', { target: { checked: true } })
-      expect(spy).toHaveBeenCalledWith(['hansel'])
+      expect(spy).toHaveBeenCalledWith('hansel')
 
       input3.simulate('change', { target: { checked: true } })
-      expect(spy).toHaveBeenCalledWith(['mugatu'])
+      expect(spy).toHaveBeenCalledWith('mugatu')
```

Comment:
Sorry for the formatting noise. Lines 226, 229, and 232 are the only changes in this file.